packages/experimental/RadioGroup/src/Radio/Radio.styling.ts

```diff
@@ -0,0 +1,81 @@
+import { radioName, RadioTokens, RadioSlotProps, RadioProps } from './Radio.types';
+import { UseStylingOptions, buildProps } from '@fluentui-react-native/framework';
+import { defaultRadioTokens } from './RadioTokens';
+
+const radioSize = 20;
+const radioRadius = radioSize / 2;
+
+const radioInnerCircleSize = 10;
+const radioInnerCircleRadius = radioInnerCircleSize / 2;
+
+export const radioSelectActionLabel = 'Select a RadioButton';
```

Comment:
We cannot have any hardcoded text as they would not get localized.

Reply:
Removed!